extensions/BugModal/web/bug_modal.js

```diff
@@ -21,7 +21,7 @@ function slide_module(module, action, fast) {
             'aria-label': is_visible ? latch.data('label-expanded') : latch.data('label-collapsed'),
         });
         if (BUGZILLA.user.settings.remember_collapsed && module.is(':visible'))
-            localStorage.setItem(module.attr('id') + '.visibility', is_visible ? 'show' : 'hide');
+            Bugzilla.Storage.set(module.attr('id') + '.visibility', is_visible ? 'show' : 'hide');
```

Comment:
Question: are there any performance implications to using this Bugzilla storage vs localstorage?

Reply:
Bugzilla.Storage is just a wrapper around localStorage so it should perform the same as localStorage
https://github.com/mozilla-bteam/bmo/blob/master/js/util.js#L848-L902